ckanext/security/templates/user/edit_user_form.html

```diff
@@ -16,3 +16,23 @@
 {% endif %}
 
 {% endblock %}
+
+{% block change_password %}
```

Comment:
It's not immediately obvious why this block is being overridden, because at first glance it's just re-adding the old and new password fields. Can a comment be added about what is changing?